genesis-beta-tester.php

```diff
@@ -23,58 +25,33 @@ function genesis_beta_tester_activation_hook() {
 
```

Comment:
Right now, when activating this with a non-Genesis theme, I get `Warning: sprintf(): Too few arguments in /wp-content/plugins/genesis-beta-tester/genesis-beta-tester.php on line 31`

At one point, we were trying to standardize the plugin compatibility messages so they were all using the same message and code and don't cause an error. https://github.com/studiopress/genesis-simple-menus/blob/develop/includes/class-genesis-simple-menus.php#L90 is working well.

Reply:
Updated https://github.com/studiopress/genesis-beta-tester/issues/3 for this issue.